internal/net/grpc/logger/logger_test.go

```diff
@@ -87,51 +102,27 @@ func Test_logger_Info(t *testing.T) {
 	type fields struct {
 		v int
 	}
-	type want struct{}
 	type test struct {
 		name       string
 		args       args
 		fields     fields
-		want       want
-		checkFunc  func(want) error
+		checkFunc  func() error
 		beforeFunc func(args)
 		afterFunc  func(args)
 	}
-	defaultCheckFunc := func(w want) error {
+	defaultCheckFunc := func() error {
 		return nil
 	}
 	tests := []test{
-		// TODO test cases
-		/*
-		   {
-		       name: "test_case_1",
-		       args: args {
-		           args: nil,
-		       },
-		       fields: fields {
-		           v: 0,
-		       },
-		       want: want{},
-		       checkFunc: defaultCheckFunc,
-		   },
-		*/
-
-		// TODO test cases
-		/*
-		   func() test {
-		       return test {
-		           name: "test_case_2",
-		           args: args {
-		           args: nil,
-		           },
-		           fields: fields {
-		           v: 0,
-		           },
-		           want: want{},
-		           checkFunc: defaultCheckFunc,
-		       }
-		   }(),
-		*/
+		{
+			name: "info success to log the message",
```

Comment:
I think the function name "info" should start with a capital case "Info".

Reply:
fixed